helm/polaris/values.yaml

```diff
@@ -518,6 +516,16 @@ realmContext:
   realms:
     - POLARIS
 
+# -- Configuration for correlation IDs.
+correlationId:
```

Comment:
Sorry for coming late to this PR, but I wonder if we may want to support standard OTel context propagation techniques instead 🤔 

https://opentelemetry.io/docs/concepts/context-propagation/